examples/websocket/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "websocket"
+version = "0.1.0"
+authors = ["Nikolay Kim <fafhrd91@gmail.com>"]
+
+[[bin]]
+name = "server"
+path = "src/main.rs"
+
+[dependencies]
+env_logger = "*"
+futures = "0.1"
+actix = "^0.3.5"
+actix-web = { git = "https://github.com/actix/actix-web.git", features=["signal"] }
```

Comment:
I am not sure this can be build on Windows

Reply:
Ok i understand ... or update actix-web : if win and feature "signal" then do nothing ?
If is not possible i allow you to remove all commit of the day (I forgot to be careful).